src/utils/resmoke/helpers.ts

```diff
@@ -1,28 +1,42 @@
-/** sdbc following by a 5 digit number
+/**
+ * sdbc following by a 5 digit number
```

Comment:
also maybe `h`, I see `h` in the regex
```suggestion
 * s/d/b/c followed by a 5 digit number
```